src/BloomExe/Book/TranslationGroupManager.cs

```diff
@@ -245,7 +245,7 @@ private static void MakeElementWithLanguageForOneGroup(XmlElement groupElement,
 		private static void StripOutText(XmlNode element)
 		{
 			var listToRemove = new List<XmlNode>();
-			foreach (XmlNode node in element.SelectNodes("descendant-or-self::*[(self::p or self::br) and not(contains(@class,'bloom-cloneToOtherLanguages'))]"))
+			foreach (XmlNode node in element.SelectNodes("descendant-or-self::*[(self::p or self::br or self::u or self::b or self::i) and not(contains(@class,'bloom-cloneToOtherLanguages'))]"))
```

Comment:
I doubt there's any way such children being removed could bother a user... might bother a template designer someday, but I doubt it.
LGTM.